app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -99,6 +99,8 @@ public class NavigationBarWidget extends UIWidget implements WSession.Navigation
     private static final int POPUP_NOTIFICATION_ID = 3;
     private static final int WEB_APP_ADDED_NOTIFICATION_ID = 4;
 
+    private static boolean disabledHeadlockForVRVideo;
```

Comment:
This must not be static, this is and attribute of the object. Regarding the wording I think I prefer a name describing what we should do, something like shouldRestoreHeadLockOnVRVideoExit.